src/audits/ImageWithoutAltText.js

```diff
@@ -27,7 +28,15 @@ axs.AuditRules.addRule({
             !axs.utils.isElementOrAncestorHidden(element);
     },
     test: function(image) {
-        return (!image.hasAttribute('alt') && image.getAttribute('role') != 'presentation');
+        var imageIsPresentational = (image.hasAttribute('alt') && image.alt == '') || image.getAttribute('role') == 'presentation';
+        if (imageIsPresentational)
+            return false;
```

Comment:
Good question. I don't think so - I think it makes sense for the scope to be "images", and role=presentation being one technique to make the rule pass. I'll tweak the text to refer to that.